ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -639,11 +640,11 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
             titleView.style = .system
         }
 
-        backgroundView.backgroundColor = style == .gradient ? .clear : color
-        standardAppearance.backgroundColor = color
+        backgroundView.backgroundColor = style == .gradient ? .clear : backgroundColor
+        standardAppearance.backgroundColor = backgroundColor
         tintColor = tokenSet[.buttonTintColor].uiColor
-        standardAppearance.titleTextAttributes[NSAttributedString.Key.foregroundColor] = shouldHideRegularTitle ? UIColor.clear : tokenSet[.titleColor].uiColor
-        standardAppearance.largeTitleTextAttributes[NSAttributedString.Key.foregroundColor] = tokenSet[.titleColor].uiColor
+        standardAppearance.titleTextAttributes[NSAttributedString.Key.foregroundColor] = systemTitleColor
+        standardAppearance.largeTitleTextAttributes[NSAttributedString.Key.foregroundColor] = systemTitleColor
```

Comment:
oo nice! bug fix we never caught?

Reply:
ya this would come up if the nav bar was set to have large titles. I didn't find a case of it but technically a bug.